packages/components/tabs/src/Tabs.doc.mdx

```diff
@@ -86,6 +86,14 @@ Use `size` prop to set the size of items.
 
 <Canvas of={stories.Size} />
 
+### keepMountedOnInactive
```

Comment:
If I understood correctly this prop is a proxy for `forceMount` prop from [Radix](https://www.radix-ui.com/primitives/docs/components/tabs#content)?
Couldn't we keep the naming then? Nothing blocking but I'm not a big fan of `keepMountedOnInactive` :sweat_smile: